packages/@dcl/playground-assets/etc/playground-assets.api.md

```diff
@@ -532,7 +532,11 @@ export type FloatArray = number[];
 // @public (undocumented)
 export const enum Font {
     // (undocumented)
-    F_SANS_SERIF = 0
+    F_MONOSPACE = 2,
+    // (undocumented)
+    F_SANS_SERIF = 0,
+    // (undocumented)
+    F_SERIF = 1

```

Comment:
```suggestion
    F_SANS_SERIF = 0,
    // (undocumented)
    F_SERIF = 1,
    // (undocumented)
    F_MONOSPACE = 2
```

Reply:
This is autogenerated code. Will break the build if changed manually